### PR TITLE
Task 4.2.2

### PR DESCRIPTION
**Pull request by ckp8**
https://docs.google.com/document/d/1IuF05cD8HyHdfLBfipF0PnwF95ExcMBJNyyRiNFWNzw/edit#task=92T0dLY3gsOI3pPK

### CODE REVIEW

**Review by Rinhammar — COMMENTED** [review]
I am not sure what has happened here. It's not showing anything in the chart. Maybe take a look at how @michaelcairney did it in this PR: https://github.com/ckp8/mashup/pull/51.

image

**Review by Rinhammar — CHANGES_REQUESTED** [review]
This is what the chart looks like for me. Missing y-axis, x-axis and all text.

image

**Comment by ckp8** [comment]
> This is what the chart looks like for me. Missing y-axis, x-axis and all text.
> 
> image

interesting, thats not how its being shown to me

File: src/pages/homePage/totalPosition/Chart.jsx [hunk 1]
```diff
@@ -144,27 +139,37 @@ const Chart = ({
         <>
```
Comment (michaelcairney) [inline src/pages/homePage/totalPosition/Chart.jsx:139]: If you get rid of the conditional render for `dimensions` should fix it

**Review by michaelcairney — COMMENTED** [review]
Good work, I think first could you merge with main and implement the YAxis and XAxis components in the components folder into the chart and also if you could remove anything that you've commented out that is no longer necessary, thanks

**Review by michaelcairney — CHANGES_REQUESTED** [review]
Just need to add 
```
tickCount={5}
showGrid={true}
hideAxisLine={true}
```
to YAxis and 
```
hideAxisLine={true}
```
to XAxis

**Review by michaelcairney — APPROVED** [review]

**Review by Rinhammar — APPROVED** [review]
Made some very minor styling changes. Otherwise, good job! Also, labels on the y-axis might need to be changed in the future, depending on what data we have then.

**Pull request merged**